test_conformance/subgroups/workgroup_common_templates.h

```diff
@@ -19,11 +19,12 @@
 #include "typeWrappers.h"
 #include <bitset>
 #include "CL/cl_half.h"
+#include "subhelpers.h"
 
 
 typedef std::bitset<128> bs128;
-inline cl_uint4 generate_bit_mask(cl_uint subgroup_local_id,
-                                  std::string mask_type,
+static cl_uint4 generate_bit_mask(cl_uint subgroup_local_id,
+                                  const char *mask_type,
```

Comment:
This change is correct - it aviods unecessary heap allocations.
However, the code below needs to be modified to build. You'll need to replace the equalities with std::strcmp(mask_type, "le") - this will restore the MacOS build (https://github.com/KhronosGroup/OpenCL-CTS/pull/1074/checks?check_run_id=1649944477)

Reply:
I've decided change to const std::string&